config/taxonomy/HOWTO_SPEC.md

```diff
@@ -0,0 +1,276 @@
+# Writing taxonomy specification
+
+The project uses schema definitions to define taxonomy.
+This document lists the requirements for writing such schemas.
+
+## What spec to follow?
+
+We use a subset of json schema DRAFT 4 that is described in this document. 
+
+You SHOULD declare the schema version in the document:
+
+```json
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    ...
+}
+```
+
```

Comment:
Perhaps mention that in order to for one file to reference a different schema file, the files should have IDs and that they should assume all schema files are mounted in a common folder.  Ex: "$id": "catalog.structs.schema.json"

Reply:
Like I wrote before, I don't think that we should add `$id` now https://github.com/IBM/the-mesh-for-data/pull/466#discussion_r606699694. BTW in Draft 4 `$id` is actually `id`. Are you sure it's necessary? Does it make sense for non published files? The spec says that it must be a URI. 

> assume all schema files are mounted in a common folder 

Will add.

Reply:
Added in the `Reference objects` section